app/components/submit-data.js

```diff
@@ -28,9 +28,6 @@ const dataMap = {
     willWithDate: 'will.isWillDate',
     willWithCodicils: 'will.codicils',
     willCodicilsNumber: 'will.codicilsNumber',
-    willCodicilsWithDate: 'will.isCodicilsDate',
-    willCodicilsDate: 'will.codicilsDate_formattedDate',
-    willDate: 'will.willDate_formattedDate',
```

Comment:
We've removed the `isWillDate` field so we could also remove `willWithDate: 'will.isWillDate',`?

Reply:
yep good spot!